.docker/Dockerfile-alpine

```
@@ -1,4 +1,4 @@
-FROM alpine:3.15.4
+FROM alpine:3.15
```

Can you revert the plain alpine versions, as those exist and 3.15 has known CVEs detected by our scanners? https://snyk.io/test/docker/alpine%3A3.15.0
```suggestion
FROM alpine:3.15.4
```